cmake/modules/FindSUNDIALS.cmake

```diff
@@ -18,9 +18,21 @@
 #
 # This module exports
 #
-#   SUNDIALS_LIBRARIES
-#   SUNDIALS_INCLUDE_DIRS
+#   SUNDIALS_LIB_IDA
+#   SUNDIALS_LIB_KINSOL
+#   SUNDIALS_LIB_PAR
+#   SUNDIALS_LIB_SER
+#   SUNDIALS_INCLUDE_DIR
 #
+# Note that, unlike most other packages, SUNDIALS headers are typically
+# installed in multiple directories; for example, a typical system installation
```

Comment:
that's fine, even if you install manually in a a prefix, you have
```
prefix/include/sundials
prefix/include/kinsol
...
```
(that's what i have in Spack)

So i would not say 

> unlike most other packages

it's just they use package name as one of the subfolders. As if we would have
```
prefix/include/dealii/dealii
prefix/include/dealii/lac
```

Reply:
Good point. I'll reword this.